src/core/js/views/drawerView.js

```diff
@@ -78,6 +80,10 @@ define(function(require) {
         // Set tabindex for select elements
         postRender: function() {
             this.$('a, button, input, select, textarea').attr('tabindex', -1);
+
+            if (this.forceCheckVisibility) {
+                this.checkIfDrawerIsAvailable();
```

Comment:
out of interest, what happens if you always run `this.checkIfDrawerIsAvailable()` here? i.e. ditch doing it on `adapt:initialize`?